lms/djangoapps/teams/static/teams/js/views/instructor_tools.js

```diff
@@ -25,14 +29,46 @@
 
                 deleteTeam: function (event) {
```

Comment:
Also add a Jasmine test that the topics page refreshes when a team deletion event is fired.